src/arch/riscv64/mod.rs

```diff
@@ -0,0 +1,15 @@
+use super::Architecture;
+
+struct Riscv64 {}
+
+pub fn new_arch() -> impl Architecture {
+    Riscv64::new()
+}
```

Comment:
Shouldn't this function be in `arch/mod.rs`?

Reply:
Yes, fixed